main.py

```diff
@@ -4957,7 +4978,7 @@ def do_command(self, command, args, sender, channel, tags, isWhisper=False):
                     try:
                         newFav = int(args[1])
                     except ValueError:
-                        self.message(channel, args[1] + " is not a number. Please try again.");
+                        self.message(channel, args[1] + " is not a number. Please try again.")
```

Comment:
Side note: Why is this using an `int()` check? Other places, e.g. `bounty <ID> <amount>`, just pretend there's a syntax error when the waifu ID isn't found or not lookupable. You do a `getWaifuById()` shortly after anyway.

Reply:
You're right, but refactors aren't really the goal of this update - I was just cleaning up low-hanging fruit reported to me by VS Code's linter.